### DISCUSSION TITLE
API broken/changed with v2.0?

### DISCUSSION THREAD

**Discussion by Marcbacca** [mealie-recipes/mealie#4425] [General]
Hi sorry, not sure what section to post this under. After the v2.0 update for Mealie, my homepage api is erroring out stating:

API Error: Invalid data
URL: http://192.168.1.5:9000/api/groups/statistics
Response Data:
<!doctype html>
<html lang="en" data-n-head="%7B%22lang%22:%7B%221%22:%22en%22%7D%7D">

The only thing changed is obviously the container update from 1.12 to 2.0.  I don't know enough to conclude if this is a change made to Mealie/api or how Homepage reads the api information. Just a heads up in case .. thanks.

**Comment by michael-genson** [comment]
Hey there,

`/groups/statistics` is removed in V2, check out the developer migration guide for more info on what changed: https://docs.mealie.io/contributors/developers-guide/migration-guide/

TL;DR you probably want `/households/statistics`

**Reply by giannis371** [reply]
Add `version: 2` in the .yaml file
```
widget:
  type: mealie
  url: http://mealie-frontend.host.or.ip
  key: mealieapitoken
  version: 2 # only required if version > 1, defaults to 1
```

**Reply by Marcbacca** [reply]
![Screenshot 2024-11-02 151901](https://github.com/user-attachments/assets/a6e8ee42-56b8-4f7d-9161-b0a5803f87ca)
This doesn't work, I had tried that but all the numbers still result in null values.

**Reply by giannis371** [reply]
Make a new api key

**Reply by Marcbacca** [reply]
> Make a new api key

Hi yes, this worked only after restarting the container a few times etc., thanks folks.
